python/device/iothub-client-sample/iothub_client_sample.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import random
+from time import sleep
+from ctypes import *
+
+timeout = c_uint(241000)
+minimumPollingTime = c_uint(9)
+receiveContext = c_uint(0)
+avgWindSpeed = 10.0
+message_count = 5
+messageHandle = [c_void_p] * message_count 
+
+IOTHUBCALLBACK = CFUNCTYPE(c_int, c_void_p, POINTER(c_int))
+SENDCALLBACK = CFUNCTYPE(c_int, c_uint, c_uint)
+
+connectionString = "HostName=mregen-python-ne-iothub.azure-devices.net;DeviceId=test;SharedAccessKey=/TKAG9mKQvzcKYS23QKDnp9L0a2ptBPQ0Da6CVGN8h0="
```

Comment:
Can you please add a comment like in the other samples regarding the connection string.